ui/src/features/WorkspaceSettings/components/MembersSettings/index.tsx

```diff
@@ -69,35 +73,48 @@ const MembersSettings: React.FC = () => {
     );
 
     if (!workspace) {
-      setError(t("Failed to add member"));
+      toast({
+        title: t("Member Could Not Be Added"),
+        description: t("There was an error when adding a new member"),
+        variant: "warning",
+      });
       return;
     }
     setEmail("");
+    setOpenMemberAddDialog(false);
   };
 
   const handleChangeRole = async (userId: string, role: Role) => {
-    setError(undefined);
     if (!currentWorkspace?.id) return;
     const { workspace } = await updateMemberOfWorkspace(
       currentWorkspace.id,
       userId,
       role,
     );
     if (!workspace) {
-      setError(t("Failed to change role of the member"));
+      toast({
```

Comment:
You have toasts in useApi as well as here now. Currently we are mostly handling toasts around gql inside of useApi, so remove here please.